### PR TITLE
User entity added

### PR DESCRIPTION
**Pull request by eeslambec**

### CODE REVIEW

File: src/main/java/uz/pdp/eufloria/domain/Address.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString
+@Table(name = "address")
+public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Address.java]: remove empty line;

File: src/main/java/uz/pdp/eufloria/domain/Basket.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package uz.pdp.eufloria.domain;
+
+import jakarta.persistence.*;
+import lombok.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Basket.java]: remove wild card '*' from imports
never ever user wild cars

File: src/main/java/uz/pdp/eufloria/domain/Orders.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package uz.pdp.eufloria.domain;
+
+import jakarta.persistence.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Orders.java]: remove wild card

File: src/main/java/uz/pdp/eufloria/domain/User.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package uz.pdp.eufloria.domain;
+
+import jakarta.persistence.*;
+import lombok.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/User.java]: remove wild card

File: src/main/java/uz/pdp/eufloria/domain/enums/Status.java [hunk 5]
```diff
@@ -0,0 +1,6 @@
+package uz.pdp.eufloria.domain.enums;
+
+public enum Status {
+    ACTIVE,
+    NON_ACTIVE
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/enums/Status.java]: change NO_ACTIVE to INACTIVE

**Comment by eeslambec** [comment]
Branch wrong opened!

**Pull request closed without merging**